internal/characters/kaveh/attack.go

```diff
@@ -0,0 +1,71 @@
+package kaveh
+
+import (
+	"fmt"
+
+	"github.com/genshinsim/gcsim/internal/frames"
+	"github.com/genshinsim/gcsim/pkg/core/action"
+	"github.com/genshinsim/gcsim/pkg/core/attacks"
+	"github.com/genshinsim/gcsim/pkg/core/attributes"
+	"github.com/genshinsim/gcsim/pkg/core/combat"
+	"github.com/genshinsim/gcsim/pkg/core/geometry"
+)
+
+var (
+	attackFrames          [][]int
+	attackHitmarks        = []int{24, 24, 26, 49}
+	attackHitlagHaltFrame = []float64{.1, .09, .09, .12}
+	attackHitboxes        = [][]float64{{2}, {2, 3}, {2}, {2, 3}}
+	attackOffsets         = []float64{0.5, -1, 0.5, -0.5}
+	attackFanAngles       = []float64{300, 360, 300, 360}
```

Comment:
cannot seem to find NA hitboxes for Kaveh (or Baizhu) in dm. not sure what right values are here